CHANGELOG.adoc

```diff
@@ -0,0 +1,12 @@
+:icons: font
+= Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on https://keepachangelog.com/en/1.0.0/[Keep a Changelog],
+and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Versioning].
+
+== [Unreleased]
+
+=== icon:archive[] Housekeeping
```

Comment:
```suggestion
=== Added
```

Reply:
Why not use some icons for the kind of change sections?

Reply:
It feels too specific and in my opinion it's more accurate/factual to say that we've added a changelog rather than we did some housekeeping.